index.html

```diff
@@ -0,0 +1,329 @@
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="head">
+      <h3><a href="index.html">My Logo</a></h3>
+        <img id="union" src="images/Union.png" />
+      <nav>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a href="#"><img id="mail" src="images/MAIL2.svg" /></a>
+          </li>
+        </ul>
+        </nav>
+    </header>
+    <main>
+      <section class="home">
+        <div class="headline">
+          <h1>Hey There. I'm Cosmas</h1>
+          <h2>I'm a software developer</h2>
+          <p>
+            I can help you build a proffessional website, feature. Look through
+            some of my work and experience and if you are pleased with my work,
+            contact me.
+          </p>
+        </div>
+        <div class="social-icons">
+          <ul>
+            <li><img src="images/Vector sv1.svg" /></li>
+            <li><img id="vec" src="images/Group icon.svg" /></li>
+            <li><img src="images/Vector sv4.svg" /></li>
+            <li><img src="images/Vector sv5.svg" /></li>
+            <li><img src="images/Vector sv6.svg" /></li>
+          </ul>
+        </div>
+        <div class="social-icons-des">
+          <ul>
+            <li><img src="images/SOCIAL1/github.svg" /></li>
+            <li><img id="vec" src="images/SOCIAL1/Linkedin icon.svg" /></li>
+            <li><img src="images/SOCIAL1/angel list.svg" /></li>
+            <li><img src="images/SOCIAL1/twitter.svg" /></li>
+            <li><img src="images/SOCIAL1/medium.svg" /></li>
+          </ul>
+        </div>
+      </section>
+      <section class="grid-container">
+        <div class="recent-works">
+          <h3>My Recent works</h3>
+          <img class="line" src="images/images-self/line (2).png" />
+        </div>
+        <div class="works-container">
+          <div class="box">
+          </div>
+          <div class="works1">
+            <div class="works-head">
+              <h3>Multi-Post Stories</h3>
+              <p class="paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has ben the industry's standard dummy text ever
+                since the 1500s,when an unknown printer took a standard dummy text.
+              </p>
+            </div>
+            <div class="languages">
+              <ul class="langul">
+                <li><a href="index.html">CSS</a></li>
+                <li><a href="index.html">HTML</a></li>
+                <li><a href="index.html">Bootstrap</a></li>
+                <li><a href="index.html">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="visit">
+              <a href="index.html">See project</a>
+            </div>
```

Comment:
- [ ] **[OPTIONAL]** Adding the correct **hover effect** to your **buttons and projects** is important for creating a more engaging and **user-friendly** website. Consider adding a color change, animation, or shape/size change to create a visually distinct hover effect that is consistent with the design guidelines in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=Px1oksowPk0aJMvW-0). 🎯